src/main/java/net/minecraftforge/event/level/ChunkEvent.java

```diff
@@ -58,9 +58,30 @@ public ChunkAccess getChunk()
      **/
     public static class Load extends ChunkEvent
     {
+        private final boolean newChunk;
+
+        @Deprecated
         public Load(ChunkAccess chunk)
+        {
+            this(chunk, false);
+        }
+
+        public Load(ChunkAccess chunk, boolean newChunk)
```

Comment:
This should be marked as internal API. (We are slowly marking all event constructors as being part of the internal API, since mods usually have no need to construct these events.)
```suggestion
        @ApiStatus
        public Load(ChunkAccess chunk, boolean newChunk)
```